docs/pages/demo.rst

```diff
@@ -187,7 +310,7 @@ launch the following in a new terminal and leave it running (``y`` is a convenie
     export UAVCAN__UDP__IFACE=127.0.0.1
     y sub --with-metadata uavcan.node.heartbeat uavcan.diagnostic.record    # You should see heartbeats
 
-Now let's see how the simple thermostat node is operating.
+Now let's see how the simple heater_voltage (Message-publisher) operates.
```

Comment:
Correct? As I understand it: `heater_voltage` is a _Message publisher_ inside the `demo_app` _node_?

Reply:
It's called simply "publisher". I don't think this is a helpful replacement as we test not just a single publisher but rather the entire node here, which is a thermostat.